### ISSUE TITLE
babelConfig.plugins.push not working ?

### ISSUE THREAD

**Issue by Romaxx** [symfony/webpack-encore#600]
Since update, I have the error:

> Module parse failed: 'import' and 'export' may only appear at the top level

I use "babel-plugin-syntax-dynamic-import" but it's like it not be used
```
 .configureBabel(function(babelConfig) {
        babelConfig.plugins.push('syntax-dynamic-import');
        babelConfig.plugins.push('transform-class-properties');
    })
```

**Comment by Kocal** [comment]
What version of Babel and Encore do you use? 
Do you have a `.babelrc` or you only configure Babel through Encore?

**Comment by Romaxx** [comment]
I use:
symfony/webpack-encore-bundle v1.6.2 
I don't have .babelrc file
I don't use yarn but npm, it's probably related to [](https://github.com/webpack/webpack/issues/8656), but since webpack-encore is not managed by npm, proposed acorn fix can't be used.

**Comment by Lyrkan** [comment]
> I don't use yarn but npm, it's probably related to webpack/webpack/issues/8656, but since webpack package is managed by webpack-encore and not by npm, proposed acorn fix can't be used.

I'm not sure that's the same issue... but if it is the different workarounds (such as doing `npm install --save-dev acorn@^6.0.5`) should work fine, even when using Encore.

Based on what you put in your first post I rather suspect that you are not using the right version of the plugins... also Encore already adds `@babel/plugin-syntax-dynamic-import` by default:

https://github.com/symfony/webpack-encore/blob/cb36619cb728d11dcb53c88c1c647251d43bb43b/lib/loaders/babel.js#L50-L55

Could you give us the content of your `package.json` file? @Kocal was asking about the version of Encore which is the JS dependency, not the bundle.

**Comment by Romaxx** [comment]
You have right about `@babel/plugin-syntax-dynamic-import` cause 'babel-plugin-syntax-dynamic-import' is not maintained (6.x) (and it not cause duplicate error so I havent see it), so I removed it. 
I have changed 'transform-class-properties' to '@babel/plugin-proposal-class-properties' because it was also deprecated.
It works with Yarn out of the box.
With npm, error remain and solution from https://github.com/webpack/webpack/issues/8656 need to be adopted to have successful build (when dynamic import is used).
